src/cml.js

```diff
@@ -460,28 +465,41 @@ class CML {
 
       if (url) return renderPr(url);
     } else {
+      let commitMessage;
       await exec(`git fetch ${remote} ${sha}`);
       await exec(`git checkout -B ${target} ${sha}`);
       await exec(`git checkout -b ${source}`);
       await exec(`git add ${paths.join(' ')}`);
-      let commitMessage = `CML PR for ${shaShort}`;
-      if (skipCi || !(merge || rebase || squash)) {
+      if (message) {
+        commitMessage = message;
+      } else {
+        commitMessage = `CML PR for ${shaShort}`;
+      }
+      if (skipCi || (!message && !(merge || rebase || squash))) {
         commitMessage += ' [skip ci]';
       }
       await exec(`git commit -m "${commitMessage}"`);
       await exec(`git push --set-upstream ${remote} ${source}`);
     }
-
-    const title = `CML PR for ${target} ${shaShort}`;
-    const description = `
-Automated commits for ${this.repo}/commit/${sha} created by CML.
-  `;
+    let description;
+    if (body) {
+      try {
+        description = (await fs.promises.readFile(body)).toString('utf8');
```

Comment:
What if your whimsical reviewer wants to use `/etc/shadow` (verbatim) as the pull request body, run CML as root, and use `md5crypt` out of sheer boredom? 😉 

Have you considered providing [`--body` and `--body-file` like `gh` does](https://cli.github.com/manual/gh_pr_create)?

Reply:
boi if you want to go down that security hole we have a whole lot of other things to take care of first 😆, I would say that since the same command basically creates an instance the executing user is trusted, or that since it requires write access to repo to trigger action that input can come from a trusted source. 

I guess we should put an asterisk and tell usesrs to not pipe untrusted user input to cml, the same warning exists for GitHub actions

Reply:
> _separating to `--body` and `--body-file` is fine with me_

That was the non-hyperbolic part of my comment. 😄 I'd prefer to keep `--body-file` as a separate option.

Reply:
:-1: to files. People can do `--body="$(cat myfile)"` quite easily.